src/ds5/ds5-device.cpp

```diff
@@ -554,6 +554,9 @@ namespace librealsense
         raw_depth_ep->register_xu(depth_xu); // make sure the XU is initialized every time we power the camera
 
         auto depth_ep = std::make_shared<ds5_depth_sensor>(this, raw_depth_ep);
+
+        depth_ep->register_info(RS2_CAMERA_INFO_PHYSICAL_PORT, filter_by_mi(all_device_infos, 0).front().device_path);
```

Comment:
`filter_by_mi()` was already called in line 545. Maybe you can keep result in var then can reuse... I couldn't find any function of `uvc_device` to get the device path, but maybe there is and then you can use `depth_devices`?